docs/src/terms.md

```diff
@@ -29,16 +29,56 @@ with the `HalfInteger` type.
 
 ```@docs
 terms
-count_terms
 ```
 
-The `L` and `S` quantum numbers are not, in general, sufficient to uniquely identify a term.
-The [`IntermediateTerm`](@ref) type allows one to specify an additional quantum number which
-would uniquely identify the term.
+## Term multiplicity and intermediate terms
+
+For subshells starting with `d³`, the possible terms may occur more
+than once (multiplicity higher than one), corresponding to different
+physical states. These arise from different sequences of coupling the
+``w`` equivalent electrons of the same ``\ell``, and are distinguished
+using a _seniority number_, which the [`IntermediateTerm`](@ref) type
+implements.
 
 ```@docs
 IntermediateTerm
 intermediate_terms
+count_terms
+```
+
+AtomicLevels.jl uses the algorithm presented in
+
+- _Alternative mathematical technique to determine LS spectral terms_
+  by Xu Renjun and Dai Zhenwen, published in JPhysB, 2006.
+
+  [doi:10.1088/0953-4075/39/16/007](https://dx.doi.org/10.1088/0953-4075/39/16/007)
+
+to compute the multiplicity of individual subshells, beyond the
+trivial cases of a single electron or a filled subshell.
+
+In the following, ``S'=2S\in\mathbb{Z}`` and
+``M_S'=2M_S\in\mathbb{Z}``, as in the original article.
+
+```@docs
+AtomicLevels.xu_terms
+AtomicLevels.Xu.X
+AtomicLevels.Xu.A
+AtomicLevels.Xu.f
+```
+
+## Term couplings
+
+To generate the possible [`terms`](@ref) of a configuration, all the
+possible terms of the individual subshells, have to be coupled
+together to form the final terms; this is done from
+left-to-right. When generating all possible [`CSFs`](@ref CSFs) from a
```

Comment:
In principle, an illustration would be really neat here I think.

Reply:
Agreed.

Reply:
Done in 9a5a945